wagtail/admin/wagtail_hooks.py

```diff
@@ -620,9 +620,14 @@ class ReportsMenuItem(SubmenuMenuItem):
     template = 'wagtailadmin/shared/menu_submenu_item.html'
 
 
+class LockedPagesMenuItem(MenuItem):
+    def is_shown(self, request):
+        return UserPagePermissionsProxy(request.user).can_unlock_pages()
```

Comment:
The RFC states that this report is available to 'users who have the "Unlock any page" permission', whereas this permission check also makes it available to ordinary editors who have pages of their own to unlock. I'd prefer to go with a strict interpretation of the RFC here - adding a new top-level menu item is a fairly significant UI change, and I'd be happier if it only happened for users in a 'management'-type role (and didn't appear/disappear depending on whether locked pages exist). Ordinary editors still get the 'your locked pages' panel on the dashboard, so they're not losing anything important this way.

Reply:
Ah okay, thanks. Will change that around in that case!